### ISSUE TITLE
Astatic semiology term raises an error

### ISSUE THREAD

**Issue by fepegar** [thenineteen/Semiology-Visualisation-Tool#105]
<img width="532" alt="Screen Shot 2020-06-22 at 14 16 50" src="https://user-images.githubusercontent.com/12688084/85292037-36c08400-b493-11ea-98b2-6c931c2f9fbd.png">

**Comment by thenineteen** [comment]
Please could you specify what patient level attributes (dominance) and semiology characteristics you used (laterality). Concretely, was it the below?

```
patient = Semiology('Astatic', Laterality.NEUTRAL, Laterality.NEUTRAL)
```

**Comment by thenineteen** [comment]
it works for me


![image](https://user-images.githubusercontent.com/10786099/85401954-c416dd80-b552-11ea-8aed-904fd19e9a03.png)

**Comment by thenineteen** [comment]
fixed in 526a465034bf5b3f08ab6e21bc755f3ec41680f6 lines 283 and 284
